### PR TITLE
multiDiffEditor: Card style + update colors

### LINKED ISSUE #202137: Multifile diff editor is all grey on start/load
* use light theme
* open a multifile diff editor for the first time
* while it loads its background is grey, much darker than the default editor background color

https://github.com/microsoft/vscode/assets/1794099/b1f314a8-64f9-4ce5-bafd-408daf4b31d2

### LINKED ISSUE #203940: multi-diff editor background is too dark
image

Can we use a lighter gray here? The black feels like a bug.

### PR DESCRIPTION
**Pull request by hbons**
Issues to fix: 
- [x] Card width is wider than viewport
- [x] Sticky header looks wrong

Preview:
image

image

Fixes #203940
Fixes #202137

### CODE REVIEW

**Review by rzhao271 — COMMENTED** [review]
Because this is endgame week, could you link an issue to this PR to help provide more context?

**Comment by hbons** [comment]
@rzhao271 forgot to mention this is for https://github.com/microsoft/vscode/issues/202137

**Review by rzhao271 — DISMISSED** [review]

**Comment by hbons** [comment]
Fixes #199333

**Comment by rzhao271** [comment]
The PR's already approved, but there's a merge conflict

**Review by TylerLeonhardt — APPROVED** [review]

File: src/vs/editor/browser/widget/multiDiffEditorWidget/colors.ts [hunk 1]
```diff
@@ -8,13 +8,13 @@ import { registerColor } from 'vs/platform/theme/common/colorRegistry';
...
 	'multiDiffEditor.headerBackground',
-	{ dark: '#808080', light: '#b4b4b4', hcDark: '#808080', hcLight: '#b4b4b4', },
+	{ dark: '#262626', light: 'tab.inactiveBackground', hcDark: 'tab.inactiveBackground', hcLight: 'tab.inactiveBackground', },
 	localize('multiDiffEditor.headerBackground', 'The background color of the diff editor\'s header')
 );
 
 export const multiDiffEditorBackground = registerColor(
 	'multiDiffEditor.background',
-	{ dark: '#000000', light: '#e5e5e5', hcDark: '#000000', hcLight: '#e5e5e5', },
+	{ dark: 'editorBackground', light: 'editorBackground', hcDark: 'editorBackground', hcLight: 'editorBackground', },
```
Comment (TylerLeonhardt) [inline src/vs/editor/browser/widget/multiDiffEditorWidget/colors.ts:17]: I think this should fix https://github.com/microsoft/vscode/issues/203940 ? cc @hediet

Comment (hediet) [reply src/vs/editor/browser/widget/multiDiffEditorWidget/colors.ts:17]: yes

**Comment by hediet** [comment]
Fixes #203940

**Pull request merged**